marlowe-runtime-web/.golden/OpenApi/golden

```diff
@@ -2186,6 +2189,26 @@
             },
             "RoleTokenConfig": {
                 "oneOf": [
+                    {
+                        "properties": {
+                            "metadata": {
+                                "$ref": "#/components/schemas/TokenMetadata"
+                            },
+                            "recipients": {
+                                "additionalProperties": {
+                                    "format": "int64",
+                                    "maximum": 1.8446744073709551615e19,
```

Comment:
Actually, the ledger may not allow a number of tokens this large. We might want to limit it to what the ledger CDDL actual provides.

Reply:
I think this is something that needs to be applied to the REST API types as a whole - basically, they are unwrapped raw data from `cardano-api`.